README.md

```diff
@@ -1,2 +1,89 @@
-# documentation
-The model documentation
+# De Re CRUD
+#### Never write a CRUD API or maintenance screen again!
+
+If you are a developer building business apps, the odds are that you
+and/or your colleagues spend too much time building and maintaining
+basic
+[CRUD](https://en.wikipedia.org/wiki/Create,_read,_update_and_delete)
+APIs and maintenance screens for all the data your app needs.  The worst
+part is that a good part of it is not even used in your app.  A
```

Comment:
This sentence is a bit confusing.  Maybe use, "The worst part is that a significant percentage of data is not put to use in your app, merely stored and displayed on command (e.g. description fields, notes, instructions, comments, ...)."